go/client/cmd_status.go

```diff
@@ -199,7 +199,7 @@ func (c *CmdStatus) outputJSON(status *fstatus) error {
 func (c *CmdStatus) outputTerminal(status *fstatus) error {
 	dui := c.G().UI.GetDumbOutputUI()
 	dui.Printf("Username:      %s\n", status.Username)
-	dui.Printf("Logged in:     %s\n", BoolString(status.LoggedInProvisioned, "yes", "no"))
+	dui.Printf("Logged in:     %s\n", BoolString(status.LoggedInProvisioned && status.PassphraseStreamCached, "yes", "no"))
```

Comment:
this might be weird since the JSON answer will show something different. Can you change the field in the struct instead?